lib/scrabble/scrabble.rb

```diff
@@ -0,0 +1,90 @@
+module Scrabble
+  class Scrabble
+
+
+    # would have preferred to nest letters in array as hash values
+    TILE_VALUES = {
+      "A" => 1,
+      "B" => 3,
+      "C" => 3,
+      "D" => 2,
+      "E" => 1,
+      "F" => 4,
+      "G" => 2,
+      "H" => 4,
+      "I" => 1,
+      "J" => 8,
+      "K" => 5,
+      "L" => 1,
+      "M" => 3,
+      "N" => 1,
+      "O" => 1,
+      "P" => 3,
+      "Q" => 10,
+      "R" => 1,
+      "S" => 1,
+      "T" => 1,
+      "U" => 1,
+      "V" => 4,
+      "W" => 4,
+      "X" => 8,
+      "Y" => 4,
+      "Z" => 10
+    }
+
+
+    def self.score(word)
+
+      stripped_word = word.gsub(/\W+/, '')
+      word_tiles = stripped_word.upcase.split(//)
```

Comment:
`.split(//)` works just fine--there is another method on String that does exactly the same thing you're trying to do here, but it doesn't need any arguments. Can you find it in the documentation? http://ruby-doc.org/core-2.2.0/String.html